assets/xml/objects/object_shn.xml

```diff
@@ -1,97 +1,125 @@
+        <DList Name="gShootingGalleryManRightFootDL" Offset="0x50D0" />
+        <DList Name="gShootingGalleryManLeftThighDL" Offset="0x5298" />
+        <DList Name="gShootingGalleryManLeftShinDL" Offset="0x53B8" />
+        <DList Name="gShootingGalleryManLeftFootDL" Offset="0x54C0" />
+
+        <!-- Textures for Shooting Gallery Man and Swamp Guide's body, as well as the Swamp Shooting Gallery Man's head -->
+        <Texture Name="gSwampShootingGalleryManNoseTLUT" OutName="swamp_shooting_gallery_man_nose_tlut" Format="rgba16" Width="16" Height="16" Offset="0x5688" />
+        <Texture Name="gShootingGalleryManTLUT" OutName="shooting_gallery_man_tlut" Format="rgba16" Width="16" Height="16" Offset="0x5888" />
+        <Texture Name="gShootingGalleryManSkin" OutName="shooting_gallery_man_skin" Format="ci8" Width="8" Height="8" Offset="0x5A88" />
+        <Texture Name="gSwampShootingGalleryManEyeOpenTex" OutName="swamp_shooting_gallery_man_eye_open" Format="rgba16" Width="32" Height="32" Offset="0x5AC8" />
+        <Texture Name="gSwampShootingGalleryManEyeHalfTex" OutName="swamp_shooting_gallery_man_eye_half" Format="rgba16" Width="32" Height="32" Offset="0x62C8" />
+        <Texture Name="gSwampShootingGalleryManEyeClosedTex" OutName="swamp_shooting_gallery_man_eye_closed" Format="rgba16" Width="32" Height="32" Offset="0x6AC8" />
+        <Texture Name="gSwampShootingGalleryManEarTex" OutName="swamp_shooting_gallery_man_ear" Format="ci8" Width="16" Height="16" Offset="0x72C8" />
+        <Texture Name="gSwampShootingGalleryManNoseTex" OutName="swamp_shooting_gallery_man_nose" Format="ci8" Width="8" Height="8" Offset="0x73C8" />
+        <Texture Name="gSwampShootingGalleryManBeardAndLipsTex" OutName="swamp_shooting_gallery_man_beard_and_lips" Format="ci8" Width="32" Height="32" Offset="0x7408" />
+        <Texture Name="gSwampShootingGalleryManHairTex" OutName="swamp_shooting_gallery_man_hair" Format="ci8" Width="16" Height="16" Offset="0x7808" />
+        <Texture Name="gShootingGalleryManHandTex" OutName="shooting_gallery_man_hand" Format="ci8" Width="16" Height="16" Offset="0x7908" />
+        <Texture Name="gShootingGalleryManBodyHairTex" OutName="shooting_gallery_man_body_hair" Format="ci8" Width="32" Height="32" Offset="0x7A08" />
+        <Texture Name="gShootingGalleryManVestTex" OutName="shooting_gallery_man_vest" Format="ci8" Width="16" Height="16" Offset="0x7E08" />
+        <Texture Name="gShootingGalleryManPantsTex" OutName="shooting_gallery_man_pants" Format="ci8" Width="8" Height="8" Offset="0x7F08" />
+        <Texture Name="gShootingGalleryManShoesTex" OutName="shooting_gallery_man_shoes" Format="ci8" Width="16" Height="16" Offset="0x7F48" />
+
+        <!-- Unused Empty Texture Animation -->
+        <TextureAnimation Name="gShootingGalleryManUnused8050TexAnim" Offset="0x8050" />
+
+        <!-- DisplayLists for an unused duplicate of the Town Shooting Gallery Man's head -->
```

Comment:
It's probably fair enough to use "unused" for the actual names here if you want, if they really don't distinguish.

Reply:
Yeah, I copied what I did for object_boss03. They look identical so there's no real way to tell them apart.

Reply:
Oh, you mean the duplicate head? Yeah, I suppose I could use "unused" for them too. It looks literally identical to the used head.